components/zendesk/zendesk.app.mjs

```diff
@@ -1,11 +1,121 @@
+import { axios } from "@pipedream/platform";
+import constants from "./constants.mjs";
+
 export default {
   type: "app",
   app: "zendesk",
-  propDefinitions: {},
+  propDefinitions: {
+    categoryId: {
+      type: "string",
+      label: "Trigger category ID",
+      description: "The ID of the trigger category. [See the docs here](https://developer.zendesk.com/api-reference/ticketing/business-rules/trigger_categories/#list-trigger-categories)",
+      async options() {
+
+        const { trigger_categories: categories } =
+          await this.listTriggerCategories({
+            params: {
+              [constants.PAGE_SIZE_PARAM]: 50,
```

Comment:
Yeah I've tested this as you pointed out but it seems like the API is not working properly here. For example I've created a number of categories from the UI (6 categories in total) and then I set the limit to 2 categories per page, then when I started to test even with `afterCursor` and `beforeCursor` I've always got only the first two categories in the list. So that's why I decided to leave it as a total of 50 categories total and I thought that would be good enough for categories. Let me know your thoughts on this one.

Reply:
For me, calling `listTriggerCategories` with the `page[after]` param worked for me to get new categories in the list.
```js
          await this.listTriggerCategories({
            params: {
              [constants.PAGE_SIZE_PARAM]: 2,
              "sort": constants.SORT_BY_POSITION_ASC,
              "page[after]": afterCursor,
            },
          });
```

If that doesn't work, I think fetching the first 50/100 categories, like you've done, would cover all/almost all actual use cases.

Reply:
My bad. I was trying only with property name `afterCursor` that was my issue! thanks!